les/server_handler.go

```diff
@@ -128,6 +128,7 @@ func (h *serverHandler) handle(p *peer) error {
 	}
 	// Reject light clients if server is not synced.
 	if !h.synced() {
+		p.Log().Debug("Light server not synced, so it's rejecting client", p)
```

Comment:
"Light server not synced, so it's rejecting client"
```suggestion
		p.Log().Debug("Light server not synced, so it's rejecting the client", p)
```